src/api/spec/controllers/attribute_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe AttributeController, type: :controller do
+  render_views
+
+  describe '#update' do
+    let(:admin) { create(:admin_user) }
+    let(:user) { create(:confirmed_user) }
+    let(:attribute) { build(:attrib_type) }
+    let(:namespace) { attribute.namespace }
+    let(:name) { attribute.name }
+
+    let(:xml_count_2) do
+      "<definition namespace='#{namespace}' name='#{name}'>
+        <count>2</count>
+        <modifiable_by user='#{user.login}'/>
+      </definition>"
+    end
+
+    let(:xml_count_3) do
+      "<definition namespace='#{namespace}' name='#{name}'>
+      <count>3</count>
+      <modifiable_by user='#{user.login}'/>
+    </definition>"
+    end
+
+    it 'will create attribute on POST' do
+      login admin
+      post :update, body: xml_count_2, format: :xml, params: { namespace: namespace, name: name }
+      expect(response.status).to eq(200)
+
+      attrib = AttribType.find_by_namespace_and_name!(namespace, name)
+      expect(attrib).not_to be_nil
+      expect(attrib.namespace).to eq(namespace)
+    end
+
+    it 'will update on POST' do
```

Comment:
I don't like that you create test data by sending it. Why don't you use the factories to create the `AttribType` with `count=2` that you update in the second step?

Reply:
because I want to test posting XML, not rendering models.

Reply:
and testing posting XML twice is better or what?

Reply:
well, you're free to write the spec yourself - I wanted to fix a bug and write a test case for it. Not to win a coding beauty contest.

Reply:
Sorry but we have some coding standards here. This is not about being beautiful but about clarity for the people who have to maintain this...
```
<norddeutsch>
Sei nich so unegal man...
</norddeutsch>
```

Reply:
¿mejor ahora?